src/libtriton/includes/triton/armOperandProperties.hpp

```diff
@@ -79,6 +81,9 @@ namespace triton {
           //! Returns the size (in bits) of the extend.
           TRITON_EXPORT triton::uint32 getExtendSize(void) const;
 
+          //! Gets subtracted flag.
+          TRITON_EXPORT bool getSubtracted(void) const;
```

Comment:
`isSubtracted()`?

Reply:
Indeed, `isSubtracted` sounds better.

Reply:
Commit 6b44a7817cbde713c2a89a4293375b902b8e9360 changes this.